Broken build after switch to `FetchContent`

[After the fix](https://github.com/jeremy-rifkin/cpptrace/commit/4ed90c1585501f99a492f296a7f177503218f235) for #171 the cmake build on stock Ubuntu 20.04 and 22.04 is broken.
Using `cmake version 3.16.3` (at least on 20.04) (the same issue was reported for 22.04)

```bash
git clone https://github.com/jeremy-rifkin/cpptrace
cd cpptrace && mkdir build && cd build
cmake ..
```

```
...
-- Cpptrace auto config: Using libgcc unwind for unwinding
-- Cpptrace auto config: Using libdwarf for symbols
-- Cpptrace auto config: Using cxxabi for demangling
CMake Error at /usr/share/cmake-3.16/Modules/ExternalProject.cmake:2555 (message):
  At least one entry of URL is a path (invalid in a list)
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/ExternalProject.cmake:3236 (_ep_add_download_command)
  CMakeLists.txt:13 (ExternalProject_Add)


-- Configuring incomplete, errors occurred!
See also "/root/cpptrace/build/_deps/zstd-subbuild/CMakeFiles/CMakeOutput.log".

CMake Error at /usr/share/cmake-3.16/Modules/FetchContent.cmake:903 (message):
  CMake step for zstd failed: 1
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FetchContent.cmake:1006 (__FetchContent_directPopulate)
  /usr/share/cmake-3.16/Modules/FetchContent.cmake:1047 (FetchContent_Populate)
  CMakeLists.txt:429 (FetchContent_MakeAvailable)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -422,9 +422,9 @@ if(CPPTRACE_GET_SYMBOLS_WITH_LIBDWARF)
       set(ZSTD_LEGACY_SUPPORT OFF)
       FetchContent_Declare(
           zstd
-          URL "${CPPTRACE_ZSTD_URL}"
-          DOWNLOAD_EXTRACT_TIMESTAMP TRUE
           SOURCE_SUBDIR build/cmake
+          DOWNLOAD_EXTRACT_TIMESTAMP TRUE
+          URL "${CPPTRACE_ZSTD_URL}"
       )
       FetchContent_MakeAvailable(zstd)
     endif()

